cmd/root.go

```diff
@@ -0,0 +1,111 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+
+	utils "github.com/fabric8-analytics/cli-tools/utils"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+var debug bool
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "crda",
+	Short: "Cli tool to interact with CRDA Platform.",
+	Long:  `Cli tool to interact with CRDA Platform. Perfoms synk token validation.`,
+	Args:  cobra.ExactValidArgs(1),
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
```

Comment:
Use log.error(err).Msg(...) 

And use log.x statements everywhere across the code. Don't use `fmt.Print`.

Reply:
Ack.